serde_derive/src/de.rs

```diff
@@ -795,9 +795,19 @@ fn deserialize_adjacently_tagged_enum(
 
     let expecting = format!("adjacently tagged enum {}", params.type_name());
     let type_name = cattrs.name().deserialize_name();
+    let deny_unknown_fields = cattrs.deny_unknown_fields();
+
+    /// If unknown fields are allowed, we pick the visitor that can
```

Comment:
Please replace these doc comments with ordinary `//` comments. This makes it look like they somehow end up in rustdoc.